### PR TITLE
RS: Add /bdbs/actions/recover REST API reference

### PR DESCRIPTION
**Pull request by rrelledge**
DOC-3849

### CODE REVIEW

File: content/operate/rs/references/rest-api/permissions.md [hunk 1]
```diff
@@ -100,6 +101,7 @@ Available management roles include:
 | <a name="view_all_users_info" style="display: block; height: 80px; margin-top: -80px;"></a>view_all_users_info | admin |
 | view_bdb_alerts | admin<br />cluster_member<br />cluster_viewer<br />db_member<br />db_viewer |
```
Comment (andy-stark-redis) [inline content/operate/rs/references/rest-api/permissions.md:102]: Found purely by chance! The links in the table to this item don't work.
```suggestion
| <a name="view_bdb_alerts" style="display: block; height: 80px; margin-top: -80px;"></a>view_bdb_alerts | admin<br />cluster_member<br />cluster_viewer<br />db_member<br />db_viewer |<a
```

File: content/operate/rs/references/rest-api/requests/bdbs/actions/recover.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+  "status": "ready"
+}
+```
+
+#### Status codes {#get-status-codes}
+
+| Code | Description |
+|------|-------------|
+| [200 OK](http://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html#sec10.2.1) | No error. |
+| [404 Not Found](http://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html#sec10.4.5) | Database UID does not exist. |
```
Comment (andy-stark-redis) [inline content/operate/rs/references/rest-api/requests/bdbs/actions/recover.md:76]: The original pages show a message saying they have been superseded. Suggested URLs are the new replacements (the new URLs are actually used in links lower in the page).
```suggestion
| [200 OK](https://www.rfc-editor.org/rfc/rfc9110.html#name-200-ok) | No error. |
| [404 Not Found](https://www.rfc-editor.org/rfc/rfc9110.html#name-404-not-found) | Database UID does not exist. |
```

File: content/operate/rs/references/rest-api/requests/bdbs/actions/recover.md [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+
+#### Required permissions
+
+| Permission name | Roles |
+|-----------------|-------|
+| [start_bdb_recovery]({{< relref "/operate/rs/references/rest-api/permissions#start_bdb_recovery" >}}) | admin<br />cluster_member<br />db_member |
+
+### Request {#post-request}
+
+The request body can either be empty or include a request body with a recovery plan.
```
Comment (andy-stark-redis) [inline content/operate/rs/references/rest-api/requests/bdbs/actions/recover.md:94]:
```suggestion
The request body can either be empty or include a recovery plan.
```

**Review by andy-stark-redis — CHANGES_REQUESTED** [review]
Few minor issues with links but the text basically looks fine.

**Comment by rrelledge** [comment]
Thanks for catching those mistakes, @andy-stark-redis!

**Review by andy-stark-redis — APPROVED** [review]
I am approve.

**Pull request merged**